src/data/auth/userServices.ts

```diff
@@ -0,0 +1,3 @@
+import { getCurrentUserMock } from '../user/__mocks/userServicesMocks';
+
+export const getCurrentUser = getCurrentUserMock;
```

Comment:
Delete this file

Reply:
You said return this file in folder 2 days ago